src/walletSdk/Anchor/Sep6.ts

```diff
@@ -53,4 +60,79 @@ export class Sep6 {
       throw new ServerRequestFailedError(e);
     }
   }
+
+  /**
+   * Deposits funds using the SEP-6 protocol. Next steps by
+   * the anchor are given in the response.
+   *
+   * @param {object} options - The options for the deposit.
+   * @param {string} options.authToken - The authentication token.
+   * @param {Sep6DepositParams} options.params - The parameters for the deposit request.
+   *
+   * @returns {Promise<Sep6DepositResponse>} Sep6 deposit response, containing next steps if needed
+   * to complete the deposit.
+   *
+   * @throws {Sep12MissingInfoError} If Sep-12 KYC info is missing for the user.
+   * @throws {Sep6DepositDeniedError} If the deposit is still being processed or not accepted.
+   * @throws {Error} If an unexpected error occurs during the deposit operation.
+   */
+  async deposit({
```

Comment:
What about exchange endoints, are they out of scope?

Reply:
ya I thought that's what this ticket is for right?

https://stellarorg.atlassian.net/browse/WAL-1060?atlOrigin=eyJpIjoiNjYzZmI4ZWExYjI3NGVmMjljYTgwYmI4MjIwMjUzZWQiLCJwIjoiaiJ9